packages/template-retail-react-app/app/components/unavailable-product-confirmation-modal/index.jsx

```diff
@@ -14,28 +14,52 @@ import {useDisclosure} from '@salesforce/retail-react-app/app/components/shared/
 import {noop} from '@salesforce/retail-react-app/app/utils/utils'
 
 /**
- * This Component will responsible to determine of a given product ids has become unavailable
- * and will prompt the users to remove them before proceeding any further
+ * This Component determines if the provided products have become unavailable or out of stock or low stock that
+ * can't be fulfilled and will prompt the users to remove them before proceeding any further
  *
  * @param productIds -  list of product ids to check for availability
+ * @param productItems -  basket product items
  * @param handleUnavailableProducts - callback function to handle what to do with unavailable products
  * @returns {JSX.Element} -  Conformation Modal Component
```

Comment:
Can we somehow make it clearer to the users that `productIds` is going to be deprecated? They're supposed to use one of the props but not both `productIds` and `productItems`.